Bio/PDB/cealign.py

```diff
@@ -52,6 +52,10 @@ def __init__(self, window_size=8, max_gap=30):
         self.max_gap = max_gap
 
         self.rms = None
+        self._rigid_motion = None
```

Comment:
I think this will be problematic if I align multiple structures to a reference. If one structure has a low RMSD, that structure's rotation/translation data will be used for all other ones. These variables should be local to the `align` function I think, just to be safe, or that function should reset them at every call.

Reply:
I don't think the scenario you described could happen because the code resets `self.rms` to `float("inf")` so the low RMSD from one alignment won't be reused, but I do think resetting the rigid motion data is a good idea. I will do that.

Reply:
I'd leave it as a local variable that is returned by optimize. It's a little annoying that the rmsd is also updated inside `_optimize`. I just worry about leftovers that cause problems with state with these variables.

Best case scenario was if the optimize function would return something like a named tuple or dataclass with the rotation/translation matrix and the rmsd, which you would then use to update the class variables but I understand if that's a big change. We also re-use the superimposer, which is again set as an instance variable, which has the same issue.

Up to you on what you think is best!